### PR TITLE
Fix return types in FITSWCSAPIWrapper and SlicedLowLevelWCS

### PR DESCRIPTION
**Pull request by Cadair**
Found while working on #9677

### CODE REVIEW

**Comment by Cadair** [comment]
4.0 bugfix

File: CHANGES.rst [hunk 1]
```diff
@@ -707,6 +707,9 @@ astropy.wcs
 - Added ``fit_wcs_from_points`` function to ``astropy.wcs.utils``. Fits a WCS
   object to set of matched detector/sky coordinates. [#9469]
 
+- ``FITSWCSAPIMixin`` now returns tuples not lists from ``pixel_to_world`` and
```
Comment (bsipocz) [inline CHANGES.rst]: isn't this technically an API change, thus belongs to the section below.

**Comment by Cadair** [comment]
I think it's now in the right place in the changelog.

**Comment by Cadair** [comment]
Please hold on merging this, looks like I have some similar fixes for `SlicedLowLevelWCS` coming.

**Pull request merged**